designer/client/src/containers/Notifications.tsx

```diff
@@ -109,16 +109,14 @@ export function Notifications(): JSX.Element {
 
                 if (!isNetworkAccess) {
                     handleChangeConnectionError("NO_NETWORK_ACCESS");
-                } else if (possibleServerNotAvailableHttpStatuses.some((status) => status === error.response.status)) {
+                } else if (possibleServerNotAvailableHttpStatuses.some((status) => status === error?.response?.status)) {
                     handleChangeConnectionError("NO_BACKEND_ACCESS");
                 } else {
                     const errorResponseData = error?.response?.data || error.message;
-
+                    console.error(`Error: ${errorResponseData}`);
                     dispatch(
                         NotificationActions.error(
                             i18next.t("notification.error.cannotFetchBackendNotifications", "Cannot fetch backend notification"),
-                            errorResponseData,
```

Comment:
In this approach, we won't display all error details which is IMO bad, what we really want to achieve here, is not to show unhandled error messages like from a reverse proxy server, where there is no control over it, but still, we want to display handled errors send from our backend server. Maybe we need to change this condition?

```
const errorResponseData = error?.response?.data || error.message;
```

to

```
const errorResponseData = error?.response?.data;
```
?